analysis/study_definition.py

```diff
@@ -1,13 +1,884 @@
+  solid_organ_transplant_nhsd_snomed=comorbidity_snomed(solid_organ_transplant_nhsd_snomed_codes),
+  solid_organ_transplant_nhsd_snomed_new=comorbidity_snomed(solid_organ_transplant_nhsd_snomed_codes_new),
+  solid_organ_transplant_nhsd_opcs4=comorbidity_ops(solid_organ_transplant_nhsd_opcs4_codes),
+  solid_organ_replacement_nhsd_opcs4=comorbidity_ops(replacement_of_organ_transplant_nhsd_opcs4_codes),
+  transplant_conjunctiva_y_code_opcs4=comorbidity_ops(conjunctiva_y_codes_transplant_nhsd_opcs4_codes),
+  transplant_ileum_1_Y_codes_opcs4=comorbidity_ops(ileum_1_y_codes_transplant_nhsd_opcs4_codes),
+  transplant_ileum_2_Y_codes_opcs4=comorbidity_ops(ileum_2_y_codes_transplant_nhsd_opcs4_codes), 
+  kidney_transplant_snomed=comorbidity_snomed(kidney_transplant_codes_ctv),
+  kidney_transplant_nhsd_opcs4=comorbidity_ops(kidney_tx_opcs4_codelist),
+  kidney_transplant_snomed_icd10=comorbidity_icd(kidney_tx_icd10_codelist),
+
+  solid_organ_transplant = patients.maximum_of("solid_organ_transplant_nhsd_snomed", "solid_organ_transplant_nhsd_snomed_new", 
+                                                    "solid_organ_transplant_nhsd_opcs4","solid_organ_replacement_nhsd_opcs4", "transplant_conjunctiva_y_code_opcs4", 
+                                                    "transplant_ileum_1_Y_codes_opcs4","transplant_ileum_2_Y_codes_opcs4", "kidney_transplant_snomed", "kidney_transplant_nhsd_opcs4"),
+ 
+  ### Neurological conditions
+  multiple_sclerosis_nhsd_snomed=comorbidity_snomed(multiple_sclerosis_nhsd_snomed_codes),
+  multiple_sclerosis_nhsd_icd10=comorbidity_icd(multiple_sclerosis_nhsd_icd10_codes),
+  motor_neurone_disease_nhsd_snomed=comorbidity_snomed(motor_neurone_disease_nhsd_snomed_codes),
+  motor_neurone_disease_nhsd_icd10=comorbidity_icd(motor_neurone_disease_nhsd_icd10_codes),
+  myasthenia_gravis_nhsd_snomed=comorbidity_snomed(myasthenia_gravis_nhsd_snomed_codes),
+  myasthenia_gravis_nhsd_icd10=comorbidity_icd(myasthenia_gravis_nhsd_icd10_codes),
+  huntingtons_disease_nhsd_snomed=comorbidity_snomed(huntingtons_disease_nhsd_snomed_codes),
+  huntingtons_disease_nhsd_icd10=comorbidity_icd(huntingtons_disease_nhsd_icd10_codes),
+  huntingtons_disease_nhsd = patients.maximum_of("huntingtons_disease_nhsd_snomed", "huntingtons_disease_nhsd_icd10"),
+  rare_neuro = patients.maximum_of("multiple_sclerosis_nhsd_snomed", "multiple_sclerosis_nhsd_icd10", 
+                                                  "motor_neurone_disease_nhsd_snomed", "motor_neurone_disease_nhsd_icd10", 
+                                                  "myasthenia_gravis_nhsd_snomed", "myasthenia_gravis_nhsd_icd10",
+                                                  "huntingtons_disease_nhsd_snomed", "huntingtons_disease_nhsd_icd10"),
+
+  ## Eligible // need to reduce eligibility of imids to those withh imid on drug/HCD
+  eligible = patients.maximum_of("downs_syndrome", 
```

Comment:
rename `eligible` to `high_risk_cohort_covid_nhsd`?

Reply:
done